### PR TITLE
Em pubsub

### PR DESCRIPTION
**Pull request by esmacik**

### CODE REVIEW

**Comment by esmacik** [comment]
* Adding comments to pubsub.ts source file from original pubsub-externs.js file for topic function, TopicBuilder class, and Message class

* Adding an @hidden tag to _json property of Message

**Comment by esmacik** [comment]
@thechenky sorry! submitted before I could put my more descriptive comments

File: src/providers/pubsub.ts [hunk 1]
```diff
@@ -35,8 +35,13 @@ export const provider = 'google.pubsub';
...
 export const service = 'pubsub.googleapis.com';
 
-/** Select Cloud Pub/Sub topic to listen to.
- * @param topic Name of Pub/Sub topic, must belong to the same project as the function.
+/**
+ * Registers a Cloud Function triggered when a Google Cloud Pub/Sub message
+ * is sent to a specified topic.
+ *
+ * @param topic The Pub/Sub topic to watch for message events.
+ *
```
Comment (thechenky) [inline src/providers/pubsub.ts]: Don't need these line breaks between tags @param and @return

File: src/providers/pubsub.ts [hunk 2]
```diff
@@ -105,15 +110,27 @@ export function _scheduleWithOptions(
...
   ) {}
 
-  /** Handle a Pub/Sub message that was published to a Cloud Pub/Sub topic */
+  /**
+   * Event handler that fires every time a Cloud Pub/Sub message is
+   * published.
+   *
+   * @param handler Event handler that runs every time a Cloud Pub/Sub message
+   *   is published.
+   *
```
Comment (thechenky) [inline src/providers/pubsub.ts]: Remove line break

**Comment by esmacik** [comment]
@thechenky I made those changes!

**Pull request merged**